components/StatementsDiscussion/StatementNodeWrapper.tsx

```diff
@@ -51,10 +43,19 @@ const StatementNodeWrapper = ({
     return focusedComment && focusedComment.id === comment.id
   }, [discussion?.comments, comment])
 
-  const FocusLink = useMemo(() => {
-    return getLinkComponent(discussion, comment)
+  const focusHref = useMemo(() => {
+    const urlObject = getFocusHref(discussion, comment)
+    return format(urlObject)
   }, [discussion, comment])
 
+  const profileHref = useMemo(
+    () =>
+      comment?.displayAuthor?.slug
+        ? `~${comment.displayAuthor.slug}`
+        : focusHref,
+    [comment?.displayAuthor?.slug, focusHref]
+  )
```

Comment:
For later: no link would be better than `focusHref` when someone is anonymous (not possible tomorrow).

Reply:
noted in Notion Todos